packages/fxa-settings/src/lib/context/context-validation.ts

```diff
@@ -0,0 +1,21 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// TODO: Figure out how to port Vat. Here's a simplistic implementation for POC.
```

Comment:
I went down a pretty big rabbit hole trying to port this, and decided I better check in with you all first. I think it's doable, but lack of types in VAT made it a pain...

Reply:
Basic validation is fine by me 👍🏽

Reply:
@vbudhram I got some basic stuff in there. I think we should file a follow up for testing it thoroughly and flushing out all the different validation types. I did my best add them as they were encountered.

Reply:
@vbudhram A follow up issue has been filed. See FXA-6800.